src/app/services/annotator/annotator.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AnnotatorService {
+  textSelection: Subject<object> = new Subject();
```

Comment:
is the `textSelection` just an `object`?

Reply:
it seems that this can be written as `textSelection = new Subject<Selection>();`